spectrum/templates/elife-xpub/initial-submission.rtf

```diff
@@ -0,0 +1,35 @@
+{\rtf1\ansi\deff3\adeflang1025
```

Comment:
this is just the original of the PDF, a pretty empty file with a title, author and abstract. Anything would do as we are not testing conversion, prepopulation of fields, or anything; only that the files can go through the browser/`nginx`/Node `app` container/Sciencebeam stack